### ISSUE TITLE
OTLP of the latest version don't send trace message to opentelemetry-collector v0.4.0

### ISSUE THREAD

**Issue by gunsluo** [open-telemetry/opentelemetry-go#874]
Describe the bug
I use the following example to send trace messages to opentelemetry collector server.(https://github.com/open-telemetry/opentelemetry-go/tree/master/example/otel-collector)

Steps to reproduce
https://github.com/open-telemetry/opentelemetry-go/tree/master/example/otel-collector

Follow the above document to configure and run the program.

What did you expect to see?

What did you see instead?
no error message

What version did you use?
version: v0.4.0
docker image I used is otel/opentelemetry-collector-contrib:0.4.0

What config did you use?
```
receivers:
  # Make sure to add the otlp receiver. 
  # This will open up the receiver on port 55680
  otlp:
    endpoint: 0.0.0.0:55680

processors:
  batch:
  queued_retry:

extensions:
  health_check: {}

exporters:
  jaeger:
    endpoint: "jaeger:14250"
    insecure: true
  otlp:
    endpoint: "otel-collector:55679"
    insecure: true
  logging:
    loglevel: debug



  logging:
      loglevel: debug

service:
  extensions: [health_check]
  pipelines:
    traces:
      receivers: [otlp]
      processors: [batch, queued_retry]
      exporters: [logging, otlp]
```

Environment
OS: macos 10.15.4

Additional context

docker image `opentelemetry-collector v0.3.0` works fine. and no error messages are print under v0.4.0.

 Hi, @MrAlias   
any idea?

**Comment by nilebox** [comment]
@gunsluo Note that OpenTelemetry Collector has just released `v0.5.0`, which includes breaking changes after updating to OTLP v0.4.0 - Go SDK has already been updated to it: https://github.com/open-telemetry/opentelemetry-go/blob/f1e3536baf79c9bd53f7f676c3bb3a9e4adf0bae/exporters/otlp/go.mod#L10

So latest releases of Go SDK and Collector should have OTLP compatibility again.

**Comment by MrAlias** [comment]
@gunsluo I'm going to close based on what @nilebox pointed out. If you upgrade and still have issues, please reopen.
